makelib/golang.mk

```diff
@@ -124,28 +118,10 @@ ifeq ($(RUNNING_IN_CI),true)
 GO_LINT_ARGS += --timeout 10m0s --out-format=checkstyle > $(GO_LINT_OUTPUT)/checkstyle.xml
 endif
 
-# NOTE: the install suffixes are matched with the build container to speed up the
-# the build. Please keep them in sync.
-
-ifneq ($(GO_PKG_DIR),)
-GO_PKG_BASE_DIR := $(abspath $(GO_PKG_DIR)/$(PLATFORM))
-GO_PKG_STATIC_FLAGS := -pkgdir $(GO_PKG_BASE_DIR)_static
-endif
-
-GO_COMMON_FLAGS = $(GO_BUILDFLAGS) -tags '$(GO_TAGS)'
-GO_STATIC_FLAGS = $(GO_COMMON_FLAGS) $(GO_PKG_STATIC_FLAGS) -installsuffix static  -ldflags '$(GO_LDFLAGS)'
+GO_COMMON_FLAGS = $(GO_BUILDFLAGS) -tags '$(GO_TAGS)' -trimpath
```

Comment:
Add `-trimpath` to Go invocations:
```
	-trimpath
		remove all file system paths from the resulting executable.
		Instead of absolute file system paths, the recorded file names
		will begin either a module path@version (when using modules),
		or a plain import path (when using the standard library, or GOPATH).
```